src/main/resources/swagger/api-docs.yaml

```diff
@@ -6726,6 +6726,61 @@ paths:
                   $ref: '#/components/schemas/ToolClass'
       x-passthrough: true
       x-passthrough-target: agora
+  /servicePerimeters/{servicePerimeterName}/projects/{projectName}:
+    put:
+      tags:
+        - Service Perimeters
```

Comment:
I'm surprised this works, but it appears to be the convention in the file 🤷‍♂️

Reply:
AFAICT this is only used to generate the section headings in the Swagger UI.  It isn't used for code generation like in AoU RW.